_posts/2017-12-11-away-week-prague/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "The importance of face-to-face in a remote world"
+title-short: Face-to-Face
+tags: [company culture, management]
+author: Anthony
+category: Inside the Plow
+permalink: /blog/2017/12/11/importance-of-face-to-face/
+description: "The role of team retreats for distributed companies"
+---
+
+![prague-scene][prague]
+
+Tucked away in a labyrinthian apartment building above a cafe in Prague, the Snowplow Analytics team met to reflect on the past year and turn our eyes to the future. We are a global team, distributed around the world, and during Away Week we come together to see old friends, greet new team members, and work together to help keep Snowplow Analytics evolving. We had team members come from the United Kingdom, France, Germany, Russia, Ukraine, Portugal, Ireland and, for the first time, the United States, all determined to turn our collective skills and abilities towards Snowplow’s success. These Away Weeks present a unique opportunity to combine our skillsets and get a shared perspective on the challenges we face as a company. Our last Away Week, for example, resulted in the release of our support for Kafka,
+
+<h2 id="culture heroes">Culture Heroes</h2>
+
+The first topic we covered was company culture. As Snowplow users, it might not be apparent what kind of benefits there are for everyone from open source users to our largest enterprise clients. Our rapid response rate and round the clock support are only made possible through the dedication of our support and services team along with the technical operations team members who overcome the technical challenges on our side so we can resolve issues for our users. Their constant vigilance and the collaborative environment at Snowplow mean we are always working to improve the Snowplow experience for our users, from a technical as well as service perspective.
+
+Learning, and professional growth, was one of the recurring themes throughout the culture conversations. Snowplow Analytics team members have incredibly diverse backgrounds, which you can read about in depth in the Team section of our company page, giving each of us a distinctive frame of mind for problem solving and insight on our daily activities. Being able to learn from such a varied group not only enhances us as individuals, but allows the sum of us to be greater than the individual parts. This Away Week was a unique experience for team members old and new: in the time that elapsed since our previous away week, the Snowplow team nearly doubled in size and, thanks in large part to our new Office Manager Nicki, we were much more organized.
+
```

Comment:
Can we maybe pull out what makes Snowplow culture unique? We can describe the session - team members got a chance to talk about what was important to them about the company culture. This is important because it isn't obvious - esp. when members are distributed across so many different geographies and time zones. The things we find that bind us are a love of learning, thirst for problem solving, and commitment to empowering our users to do more with their data. There's a strong streak of "data and technology optimism" that runs through the company and is one of the many things that motivate our team. The other factor that came through strongly was the autonomy that each team member has, the freedom and the space to innovate and including making mistakes, where necessary.